addons/hive.yml

```diff
@@ -0,0 +1,11 @@
+---
+- name: install hive
+  hosts: frontendnodes
+  become: yes
+  roles:
+    - role: java
+    - role: mysql
+      when: HIVE_METASTORE == 'mysql'
```

Comment:
what happens if the mysql role is not applied (ie if `HIVE_METASTORE` is something else?